cubist/tests/test_cubist.py

```diff
@@ -24,69 +25,103 @@ def test_model_instance(expected_output):
 def test_n_rules():
     model = Cubist(n_rules=500)
     model.fit(X, y)
-    assert model.is_fitted_
+    check_is_fitted(model)
 
 
 def test_n_rules_fail():
     model = Cubist(n_rules=10000000)
     with pytest.raises(ValueError):
         model.fit(X, y)
-        assert model.is_fitted_
+        check_is_fitted(model)
 
 
 def test_n_committees():
     model = Cubist(n_committees=5)
     model.fit(X, y)
-    assert model.is_fitted_
+    check_is_fitted(model)
 
 
 def test_n_committees_fail():
     model = Cubist(n_committees=101)
     with pytest.raises(ValueError):
         model.fit(X, y)
-        assert model.is_fitted_
+        check_is_fitted(model)
```

Comment:
Why does the first set of tests pass but the second doesnt?

Reply:
so the R package suggests the max number of committees is 100 but the C code might not even require that, will test

Reply:
Yes I noticed the checks after commenting.